spec/models/user_spec.rb

```diff
@@ -172,4 +172,27 @@
       expect(another_user.can_receive_deposits_from.to_a).to eq [@subject]
     end
   end
+  describe "class methods" do
+    describe "recent_users" do
+      let(:new_users) { User.all.order(created_at: :desc) }
+
+      before do
+        (1..3).each {|i| User.create(email: "abc#{i}@blah.frg", password: "blarg1234", created_at:DateTime.now - i.days) }
+      end
+
+      it "returns based on start date" do
+        results = User.recent_users(Date.today - 2.days)
```

Comment:
cam156: Yeah, the context is "calling with one argument", that is more clear to me. It describe what changes.